spec/swagger_helper.rb

```diff
@@ -13,6 +13,7 @@
   config.before(:each) do
     stub_request(:get, /#{Settings.rbac_url}/)
       .to_return(status: 200,
+                 headers: { 'Content-Type' => 'application/vnd.api+json' },
```

Comment:
How is this part of the scope of the PR?

Reply:
Unit tests are failing when the header is not set. See https://github.com/RedHatInsights/insights-rbac-api-client-ruby/pull/11.

Reply:
Ideally compensations should be marked with a comment and a FIXME to get it removed after the bug is fixed on the other side.